test/runtests.jl

```diff
@@ -1,3 +1,12 @@
+# Working around https://github.com/JuliaLang/Pkg.jl/issues/2500 for CI
+if VERSION <= v"1.8-"
+    test_project = first(Base.load_path())
+    preferences_file = "LocalPreferences.toml"
```

Comment:
This relative path is pretty fishy.  `isfile(preferences_file)` below is almost always false.

Reply:
I stole this from another project where it worked xD